src/cpp/session/include/session/SessionOptions.hpp

```diff
@@ -309,6 +309,16 @@ class Options : boost::noncopyable
       return allowOverlay() || allowRpubsPublish_;
    }
 
+   bool allowExternalPublish() const
```

Comment:
When doing this I'd remove the reference to allow-rpubs-publish (but of course keep it in code for backward compatibility)